WordPress/Classes/System/RootViewCoordinator.swift

```diff
@@ -87,6 +87,18 @@ class RootViewCoordinator {
         self.rootViewPresenter = nil
     }
 
+    func showPostSignUpTabForNotSites() {
```

Comment:
There might be a minor typo here with the name. Is it supposed to be this?
```suggestion
    func showPostSignUpTabForNoSites() {
```

Reply:
Nice catch! Fixed in e5b010b298bba6530a35ee92ebb39acc664c2ef1